extensions/amp-analytics/amp-analytics.md

```diff
@@ -442,7 +442,7 @@ NOTE: There is a [known issue](https://github.com/ampproject/amphtml/issues/1089
 
 The `visibilitySpec` is a set of conditions and properties that can be applied to `visible` or `hidden` triggers to change when they fire. If multiple properties are specified, they must all be true in order for a request to fire. Configuration properties supported in `visibilitySpec` are:
   - `waitFor`: This property indicates that the visibility trigger should wait for a certain signal before tracking visibility. The supported values are `none`, `ini-load` and `render-start`. If `waitFor` is undefined, it is defaulted to [`ini-load`](#initial-load-trigger) when selector is specified, or to `none` otherwise.
-  - `reportWhen`: This property indicates that the visibility trigger should wait for a certain signal before sending the trigger. The only supported value is `documentExit`. `reportWhen` and `repeat` may not both be used in the same visibilitySpec.
+  - `reportWhen`: This property indicates that the visibility trigger should wait for a certain signal before sending the trigger. The only supported value is `documentExit`. `reportWhen` and `repeat` may not both be used in the same visibilitySpec. When `reportWhen` is specified, the `visibilitySpec` may include the `forceReport` property to indicate that the report should be sent at the time of the signal even if visibility requirements have not been met. In that case, any relevant variables (`totalVisibleTime`, etc.) will default to 0.
```

Comment:
Question: I don't believe the `totalVisibleTime` will default to 0 with `forceReport`. Thought the idea of introducing `forceReport` was to collect the totalVisibleTime above certain percentage.

Reply:
It defaults to 0 in local testing.

The goal of `forceReport` is to be able to report `totalVisibleTime` even when its zero. We usually set `visiblePercentageMin` to 50% but we want to send pings (with `totalVisibleTime=0`) when zero pixels were on screen, 40% of pixels were on screen, etc.

Reply:
Right, but it doesn't default to 0. I believe it should still be the time that's between the specified visibility percentage. 

When you say it defaults to 0 in local test? What is the expected `totalVisibleTime`? Is it true that the element hasn't be 50% visible before?